run_evaluation.py

```diff
@@ -48,4 +48,45 @@
 eval.save_stacked_evaluation_heatmaps(h, X, y, stacked_predict_heatmaps_file, stacked_ground_truth_heatmaps_file, filename)
 print(f"Saved stacked evaluation heatmaps as {filename} to disk")
 
+# %% Plot predicted keypoints from heatmaps on bounding box image
+import numpy as np
+from HeatMap import HeatMap
+
+generator = DataGenerator(
+            df=val_df,
+            base_dir=DEFAULT_VAL_IMG_PATH,
+            input_dim=INPUT_DIM,
+            output_dim=OUTPUT_DIM,
+            num_hg_blocks=DEFAULT_NUM_HG,
+            shuffle=False,  
+            batch_size=1,
+            online_fetch=False)
+
+# Select image to predict heatmaps
+X_batch, y_stacked = generator[168] # choose one image for evaluation
+y_batch = y_stacked[0] # take first hourglass section
+X, y = X_batch[0], y_batch[0] # take first example of batch
+
+# Get predicted heatmaps for image
+predict_heatmaps=eval.predict_heatmaps(h, X)
```

Comment:
This may be the one area that conflicts with my branch, predict_heatmaps will take in a batch instead of a single example. I think it would be easiest if we merge your branch first and I fix this line in my branch after rebasing.